rllab/misc/tensorboard_summary.py

```diff
@@ -0,0 +1,142 @@
+from rllab.misc.console import mkdir_p
```

Comment:
Please reorder imports to conform to PEP8

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
It seems the order is fine now.

Reply:
I have changed the order. System library goes first, the second one is the third-party library, and the last one is our library.

>import os
>import tensorflow as tf
>from rllab.misc.console import mkdir_p

Reply:
These are ordered correctly, but need to be separated into groups:

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.
> 
> Imports should be grouped in the following order:
> 
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
>
> __You should put a blank line between each group of imports.__

```python
import os

import tensorflow as tf

from rllab.misc.console import mkdir_p
```